horizons/world/managers/productionfinishediconmanager.py

```diff
@@ -152,4 +154,4 @@ def get_resource_string(self, instance, res):
 		This key MUST be unique!
 		"""
 		return "produced_resource_" + str(res) + "_" + str(instance.position.origin)\
-		       + "_" + str(Scheduler().cur_tick)
+		       + "_" + str(uuid.uuid1())
```

Comment:
I'd rather user `uuid.uuid4()` then: "Generate a random UUID." The "produce_resource" prefix and the resource might help in debugging, so I would keep that.